atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -47,15 +49,13 @@ public File acceptorLogDir() {
     public abstract String localServer();
 
     @Size(min = 1)
-    public abstract Set<String> leaders();
+    public abstract List<String> leaders();
 
     @JsonProperty("lockCreator")
     @Value.Default
     public String lockCreator() {
-        if (leaders().isEmpty()) {
-            throw new IllegalStateException("The leaders block cannot be empty");
-        }
-        return leaders().iterator().next();
+        Preconditions.checkState(!leaders().isEmpty(), "The leaders block cannot be empty");
+        return Iterables.getFirst(leaders(), null);
```

Comment:
Flag: This change could cause the same `LeaderConfig` to return a different lockCreator (i.e. Iterables.getFirst might return something other than iterator().next() did). 

I can't think of why this would cause an issue, unless different nodes run different versions of AtlasDB (which I don't think ever happens).  So, this is probably fine, but let me know in case I'm wrong.